src/main/java/com/heatheryou/httpserver/Main.java

```diff
@@ -4,31 +4,27 @@
 import com.heatheryou.httpserver.route.handler.BuildResponse;
 import com.heatheryou.httpserver.route.handler.ResponseBuilder;
 
-import java.io.IOException;
-import java.util.Arrays;
-
 public class Main {
-    public static void main(String[] args) throws IOException {
-        if (args.length != 1) {
-            System.out.println(Arrays.toString(args));
-            System.err.println("Usage: java HTTPServer <port number>");
-            System.exit(1);
+    public static void main(String[] args) throws Exception {
+        ISystemOutput systemOutput;
+        systemOutput = new SystemOutput();
```

Comment:
I think it's more idiomatic to just do this on one line: `ISystemOutput systemOutput = new SystemOutput();`.

Reply:
✔️